plugins/techdocs-react/api-report.md

```diff
@@ -70,6 +70,10 @@ export interface TechDocsApi {
   ): Promise<TechDocsEntityMetadata>;
   // (undocumented)
   getTechDocsMetadata(entityId: CompoundEntityRef): Promise<TechDocsMetadata>;
+  // (undocumented)
+  issueUserCookie(): Promise<{
+    expiresAt: string;
```

Comment:
could potentially have been just Date as well, and letting the http layer encode it as a string